generators/app/package.json

```diff
@@ -2,18 +2,22 @@
   "name": "polymer-starter-kit-custom-build",
   "version": "0.0.0",
   "description": "A starting point for Polymer 1.0 custom build apps",
-  "license": "http://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
```

Comment:
yeah I know it's different. I've been told that Polymer repositories have to be consistent by our open source office, so I'm making sure we use the same approach as the other projects. @azakus does this look right to you? package.json says BSD-Clause3, and bower.json links to LICENSE.txt?